src/views/conference/2020/index/index.jsx

```diff
@@ -32,9 +32,6 @@ const ConferenceSplash = () => (
                     <strong><FormattedMessage id="conference-2020.desc1a" /></strong>
```

Comment:
I notice in the google doc, this is on a new line, but isn't here. Not sure if that's intentional.